### ISSUE TITLE
[Bug]: 

### ISSUE THREAD

**Issue by jaredscarr** [nsccs/club_website#49]
### What happened?

The new functionality for the Minecraft page is great, but the full screen map presents a UX problem. On both my 15" laptop and standard sized iphone14 you can scroll down to the map and effectively be stuck at that point. Stuck meaning the map has taken up the entire screen and thus all swiping/scrolling only works on the map and the rest of the site can no longer be accessed.

### If you expected something else to occur what did you expect?

I expect to be able to navigate as normal.

### What operating system are you using?

Windows 11 with 15 screen and iOS with standard screen size.

### What browsers are you seeing the problem on?

Firefox, Chrome, DuckDuckGo, Edge

### Relevant log output

```shell
N/A
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jaredscarr** [comment]
I think one solution could be to give the map a container and margin or padding to always expose some edge of the site to allow for normal navigation.

**Comment by jaredscarr** [comment]
It looks like with v9 the call to initialzeResources is expecting a Device, but is being passed a [context](https://github.com/visgl/deck.gl/blob/9947ddd902475407ea53d05a1ee94422fd335bf1/modules/arcgis/src/deck-layer-view-2d.js#L17C12-L17C42) . I have found the [upgrade guide](https://github.com/visgl/luma.gl/blob/509787e340f140c58f5621fcdfe5e7913677061a/docs/api-reference/core/device.md?plain=1#L21) which shows how to create a Device. I attempted the directions to attach the context to a new Device, but get an error that Device.attach is not a function.

```javascript
const gl = this.context;
const device = Device.attach(gl);
```
It seems like the issue is, we need to take the gl context that ArcGIS creates, and convert it/wrap it into a device. Is that correct? If so, are there any examples in the v9 codebase of how the gl context is created/how the device is created?
